modules/php/Cards/Dodge.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace BANG\Cards;
+
+use BANG\Models\BrownCard;
+
+class Dodge extends BrownCard
+{
+  public function __construct($id = null, $copy = '')
+  {
+    parent::__construct($id, $copy);
+    $this->type = CARD_DODGE;
+    $this->name = clienttranslate('Dodge');
+    $this->text = clienttranslate('Discard to avoid an attack and draw one card'); // TODO official description
+    $this->symbols = [[SYMBOL_MISSED, SYMBOL_DRAW], [SYMBOL_DRAW]]; // TODO just test how it will look like
+    $this->copies = [
+      BASE_GAME => [],
+      HIGH_NOON => [],
+      DODGE_CITY => ['7D', 'KH'],
+    ];
+    $this->effect = ['type' => DEFENSIVE];
+  }
+
+  public function playCard($player)
```

Comment:
TODO this works but draw executes first so player has dodge and also new card on hand for a while